sdk/core/core-http/lib/util/sanitizer.ts

```diff
@@ -72,6 +72,14 @@ export class Sanitizer {
   }
 
   private replacer(key: string, value: unknown) {
+    // Ensure Errors include their interesting non-enumerable members
+    if (value instanceof Error) {
+      return {
+        ...value,
+        name: value.name,
+        message: value.message
```

Comment:
I am curious about why we still need to explicitly set name and message. I thought `...value` would do it any way (except we might not have name/message on the `value`

Reply:
the expansion `...value` only pulls out enumerable properties.

Reply:
Got it. Time for me to learn about properties.